internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -160,14 +185,17 @@ func (s *Server) ApplyConfiguration(ctx context.Context, in *machine.ApplyConfig
 		return nil, err
 	}
 
-	switch {
-	// --immediate
-	case in.Immediate:
+	//nolint:exhaustive
```

Comment:
to be safe, we could probably add an option `default:` which returns an error (it should never be reached, but still)